src/objective-c/tests/ThirdPartyTests/Libuv/BUILD

```diff
@@ -0,0 +1,24 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application", "ios_unit_test")
+load("@build_bazel_rules_apple//apple/testing/default_runner:ios_test_runner.bzl", "ios_test_runner")
+
+
+ios_test_runner(
+    name = "ios_x86_64_sim_runner",
+    device_type = "iPhone 8",
+)
+
+objc_library(
+    name = "LibuvTestsLib",
+    testonly = True,
+    srcs = glob(["*.m"]),
+    deps = [
+        "//external:uv"
```

Comment:
I think it's worth noting in this file that the iOS tests are being run against gRPC's version of libuv, and not necessarily the same version of libuv that gets packaged for iOS releases.

Reply:
thanks, updated PR description w/ more details